src/bluray_com.py

```
@@ -213,11 +211,11 @@ async def extract_bluray_release_info(html_content: str, meta: Meta) -> list[Rel
     try:
         base_dir = meta.base_dir
         uuid = meta.uuid
-        debug_path = Path(base_dir) / "tmp" / uuid / f"debug_bluray_{release_type}.html"
+        debug_path = Path(base_dir) / "tmp" / uuid / f"debug_bluray_{release_type}_{product_id}.html"
```

Ruff (0.15.21)

[error] Undefined name `product_id`

(F821)